src/actions/secrets.js

```diff
@@ -80,6 +111,15 @@ export function createSecret(postData, namespace) {
     } catch (error) {
       dispatch({ type: 'SECRET_CREATE_FAILURE', error });
     }
+    const serviceAccount = await getServiceAccount(
+      postData.metadata.labels.serviceAccount,
+      namespace
+    );
+    patchServiceAccount(
```

Comment:
should we be waiting for / returning this?

Reply:
We must wait for getServiceAccount as it is needed for the patching.

Reply:
yes but I mean should we be waiting for the patch operation to complete before returning from this API function? or return the promise from the patch API so the consumer of `createSecret` can wait. As it stands, this function will return immediately once `patchServiceAccount` is called, and will not wait for it to complete, whether it succeeds / fails.

Reply:
done 😸

Reply:
I'm not seeing the change here